KAHN_SQL_EXC_Answers.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<head>
+	<style>
+		h1{
+			text-align: center;
+			font-style: bold;
+			font-family: sans-serif;
+		}
+	
+		p{
+			font-family: sans-serif;
+
+		}
+		</style>
```

Comment:
General consensus *seems* to be to use a seperate style file, no matter how few changes one makes, and then use 
```
<style src='.style.css'>
```
to include those style changes.